py_gtfs_rt_ingestion/pyproject.toml

```diff
@@ -6,7 +6,8 @@ authors = ["MBTA CTD <developer@mbta.com>",
            "Mike Zappitello <mzappitello@mbta.com>"]
 
 [tool.poetry.dependencies]
-python = "^3.7"
+python = "^3.9"
+pyarrow = "^8.0.0"
```

Comment:
update python and add pyarrow to our list of dependencies.

i used to add pyarrow `poetry add pyarrow` on the command line. i think that edits the dot toml file and also invokes `poetry install` which uses `pip` to add pyarrow to the virtualenv, also installing pyarrows dependencies, and adding all of them to the poetry dot lock file.